docs/generators/endpoint.md

```diff
@@ -8,7 +8,8 @@ Usage:
 
 Options:
   -h,   --help               # Print the generator's options and usage
-        --skip-cache         # Do not remember prompt answers           Default: false
+        --skip-cache         # Do not remember prompt answers             Default: false
+        --skip-install       # Do not automatically install dependencies  Default: false
```

Comment:
Strange. That might be coming from Yeoman. Anyway, it won't have any affect on the endpoint generation, so there's no point in documenting it.